README.md

```diff
@@ -252,7 +253,7 @@ Have a look at the [change log](./CHANGELOG.md) for recent updates.
 ### 1.0.0
 
 - support optimistic write support through attributes with conflict resolution
-  (CRDT-like)
+  (CRDT-like) -> Did we do this?
```

Comment:
No, I think you could move the roadmap to a separate file and remove the 1.0.0

Reply:
Done